recipe_manager.py

```diff
@@ -128,8 +128,9 @@ def save_recipe(self):
 
         json_file_name = self.create_py_dict()
 
-        with open(json_file_name + ".json", "w") as f:
-            json.dump(self.save_pydict, f)
+        if len(json_file_name) > 0:
```

Comment:
what do these file names look like?

Reply:
Each file is named after the input of the 'name' textbox. At this point, duplicating name field entries will overwrite files. We may change that behavior, as we have multiple chocolate chip cookie recipes :)